virttest/utils_v2v.py

```diff
@@ -209,6 +209,7 @@ def __init__(self, test, params, env):
         self.delete_vm = 'yes' == params.get('vm_cleanup', 'yes')
         self.virsh_session_id = params.get("virsh_session_id")
         self.windows_root = params.get("windows_root", "C:\WINDOWS")
+        self.output_method = params.get("output_method")
```

Comment:
let us declare the param in cfg with the comments explaining the usage.

Reply:
I have updated the cfg file for describing the usage of output_method. Please have a look, Thanks!